security/password_migration.rst

```diff
@@ -102,6 +102,13 @@ using the ``migrate_from`` option:
             ],
         ]);
 
+With this setup:
+
+* New users will be encoded with the new algorithm;
+* Whenever a user logs in whose password is still stored using the old algorithm,
+  Symfony will verify the password with the old algorithm and then re-encode
```

Comment:
```suggestion
  Symfony will verify the password with the old algorithm and then rehash
```

Confusing Symfony terminology, but the password encoders are doing hashing. This is also consistent with earlier uses of "rehash(ing)" in the beginning of this chapter.

Reply:
I made this change while merging.